working directory:  /home/megu/ECoG_CNNs
omp thread num:  1
ncpus:  1
cuda visible devices:  GPU-1ce72701-f930-8001-09eb-a082cba45729
Traceback (most recent call last):
  File "main.py", line 146, in <module>
    main()
  File "main.py", line 43, in main
    net.load_state_dict(torch.load(model_path))
  File "/home/megu/.pyenv/versions/3.7.9/lib/python3.7/site-packages/torch/serialization.py", line 581, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/megu/.pyenv/versions/3.7.9/lib/python3.7/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/megu/.pyenv/versions/3.7.9/lib/python3.7/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/home/megu/ECoG_CNNs/Result/MK18_expt.6_1/model.pth'
start time:  1635257984
end time:  1635264729
run time:  6745
